openapi.yml

```diff
@@ -0,0 +1,578 @@
+openapi: 3.0.0
+info:
+  description: Preservation Catalog HTTP API
+  version: 1.0.0
+  title: Preservation Catalog HTTP API
+  license:
+    name: Apache 2.0
+    url: 'http://www.apache.org/licenses/LICENSE-2.0.html'
+servers:
+  - url: 'https://preservation-catalog-{env}-01.stanford.edu'
+    description: Production service
+    variables:
+      env:
+        default: prod
+  - url: 'https://preservation-catalog-{env}-01.stanford.edu'
+    description: Staging service
+    variables:
+      env:
+        default: stage
+  - url: 'https://preservation-catalog-{env}-01.stanford.edu'
+    description: QA service
+    variables:
+      env:
+        default: qa
+tags:
+  - name: catalog
+    description: Add and update MOABs in the catalog
```

Comment:
Moab is not an acronym;  it should be Moab everywhere, if we care.

Reply:
Fixed!